.env.wordpress.local

```diff
@@ -0,0 +1,2 @@
+NODE_ENV=production
```

Comment:
I would normally say don't commit this file - but will leave it for now since it doesn't reveal any privileged information and won't vary too much.

Reply:
Okay that's fine it's probably why the file was being ignored by github originally but if it doesn't vary too much thats okay :)